docs/vmanomaly.md

```diff
@@ -0,0 +1,126 @@
+
+   Simplistic model, that detects outliers as all the points that lie farther than a certain amount
+   from time series mean (straight line). Keeps only two model parameters internally:
+   `mean` and `std` (standard deviation).
+
+2. **Prophet**
+
+   _(simplest in configuration, recommended for getting starting)_
+
+   Uses Facebook Prophet for forecasting. Anomaly score is computed of how close the actual time
+   series values follow the forecasted values (yhat), and whether it’s within forecasted bounds
+   (_yhat_lower_, _yhat_upper_). Anomaly score reaches 1.0 if the actual data values equal to
+   _yhat_lower_ or _yhat_upper_. Anomaly score is above 1.0 if the actual data values are outside
+   the _yhat_lower_/_yhat_upper_ bounds.
+
+   See Prophet documentation: https://facebook.github.io/prophet/
+
+3. **Holt-Winters**
+
+   Very popular forecasting algorithm. See [statsmodels.org documentation](
+   https://www.statsmodels.org/stable/generated/statsmodels.tsa.holtwinters.ExponentialSmoothing.html)
+   for Holt-Winters exponential smoothing.
+
+4. **Seasonal-Trend Decomposition**
+
+   Extracts three components: season, trend and residual, that can be plotted individually for
+   easier debugging. Uses LOESS (locally estimated scatterplot smoothing).
+   See [statsmodels.org documentation](https://www.statsmodels.org/dev/examples/notebooks/generated/stl_decomposition.html)
+   for LOESS STD.
+
+5. **ARIMA**
+
+   Commonly used forecasting model. See [statsmodels.org documentation](https://www.statsmodels.
+   org/stable/generated/statsmodels.tsa.arima.model.ARIMA.html) for ARIMA.
+
+6. **Rolling Quantile**
+
+   Simple moving window of quantiles.
+
+
+### Examples
+For example here’s how Prophet predictions could look like on example of real data  
+(Prophet auto-detected seasonality interval):
+![prophet](anomaly/prophet-example.png)
+
+And here’s how Holt-Winters predictions real-world data could look like (seasonality manually 
+ set to 1 week). Notice that it predicts anomalies in 
+different places than Prophet, because the model noticed there are usually spikes on Friday 
+morning, so it accounted for that:
+![hw](anomaly/holtwinters-example.png)
+
+## Process
+Upon starting, vmanomaly queries the initial range of data, and trains its model (“fit” by convention).
+
+Then, reads new data from VictoriaMetrics, according to schedule, and invokes its model to compute 
+“anomaly score” for each data point. The anomaly score ranges from 0 to positive infinity. 
+Values less than 1.0 are considered “not anomaly”, values greater or equal than 1.0 are 
+considered “anomalous”, with greater values corresponding to larger anomaly.
+Then, VMAnomaly pushes the metric to vminsert (under user-configured metric name, 
+optionally preserving labels).
+The writer is also pluggable, users can supply their own writers.
+
+
+## Usage
+Script accepts only one parameter -- config file path:
+
+```sh
+python3 vmanomaly.py config_zscore.yaml
```

Comment:
Can we add config file examples? Or some description for config file.

Reply:
See PR description